backend/src/main/java/de/bund/digitalservice/ris/caselaw/domain/DocumentationUnitService.java

```diff
@@ -488,6 +490,30 @@ private void saveForRecycling(Documentable documentationUnit) {
     }
   }
 
+  @Transactional(rollbackFor = BadRequestException.class)
```

Comment:
Wenn also eine zuweisung einer einzelenen docunit scheitert, scheitert der ganze vorgang, da die ganze transaction dann scheitert oder? Sollte man vielleicht nicht nur die einzige docunit scheitern lassen und das der userin zurückschicken?

Reply:
Das hatten wir explizit mal bei einem Refinement so besprochen, dass das einfacher ist, weil wir uns sonst in der UI ja überlegen müssen, wie wir kenntlich machen, was jetzt genau gefailed ist. Es sollte ja eigentlich auch gar nicht auftreten. 😬 Im Zweifelsfall muss man es dann einfach insgesamt nochmal ausführen.

Reply:
ja, ich erinnere mich wieder, dass wir das besprochen hatten